rollup.config.js

```diff
@@ -19,10 +19,11 @@ export default {
 		format: 'iife',
 		sourcemap: production,
 	},
+	inlineDynamicImports: true,
```

Comment:
instead of this we could add
  ```"routify": {
    "dynamicImports": false
  }```
to the package.json. Not sure which is preferable.

Reply:
I decided to go with the suggestion made in the issue [here](https://github.com/roxiness/routify.dev/issues/218)